cluster/addons/dns/coredns/coredns.yaml.base

```diff
@@ -129,6 +129,8 @@ spec:
         - name: config-volume
           mountPath: /etc/coredns
           readOnly: true
+        - name: tmp
```

Comment:
what file does coredns use from the host /tmp ?

Reply:
from the individual commit, i can assume this is for the logs.
this seem a bit hack especially this is for master code.
normally the pod app should be able to get its log out. we should leverage, or use the termination log if this for pod crash csaes.

Reply:
This is a cherry-pick from community Kubernetes. https://github.com/kubernetes/kubernetes/pull/82128
we do have this kind of error before this fix.

Reply:
Thanks for the info.

This seems to be temp workaround for an issue in klog in k8s release 1.14 and 1.15 as the change only short-lived in those two releases only, in 1.16 k8s didn't take this change.

we probably should not take it instead check how eventually the fix was addressed.

is this change helping us?  the root issue is the coredns cannot access the api server and hence the crash-restart, if the root issue is addressed, we won't need this fix -- i.e. we can afford even a short restart of the Dns service for NOW.

Reply:
at least for now, it help us. once the root cause of cordons addressed, not sure what will happened once we remove this.

Reply:
as we discussed, let's add some comment on this change for future update to coredns in arktos if resource is available.